bin/problem.py

```diff
@@ -916,38 +918,104 @@ def validate_data(problem, mode: validate.Mode, constraints: dict | bool | None
                 log(f"Not running answer_validators for{msg} problems.")
             return True
 
+        action = (
+            "Invalidation"
+            if mode == validate.Mode.INVALID
+            else (
+                f"Collecting {mode} constraints" if constraints else f"{mode} validation"
+            ).capitalize()
+        )
```

Comment:
Why `.capitalize()` the `action` when both branches return something that already starts with a capital? :thinking: 

Oh wait, the `mode` is not capitalized. Maybe this can be simplified to only `mode.capitalize()` then :slightly_smiling_face:

Reply:
I just moved this, so no idea what exactly is going on XD